_schema.graphql

```diff
@@ -1119,6 +1119,46 @@ type ArtworkContextFair {
 
   # The exhibitors with booths in this fair.
   exhibitors_grouped_by_name: [FairExhibitorsGroup]
+
+  # Artworks Elastic Search results
+  filteredArtworks(
+    acquireable: Boolean
```

Comment:
I wish we created an interface for this set of filter params because we have it duplicated in a lot of places now.